pkg/config/datafileprojectconfig/config.go

```diff
@@ -82,6 +82,37 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
 	return c.attributeKeyToIDMap[key]
 }
 
+// GetAttributes returns attributes slice
+func (c DatafileProjectConfig) GetAttributes() []entities.Attribute {
+	attributes := make([]entities.Attribute, 0, len(c.attributeMap))
+	for _, attr := range c.attributeMap {
+		attributes = append(attributes, attr)
+	}
+	return attributes
+}
+
+// GetEvents returns events slice
+func (c DatafileProjectConfig) GetEvents() []entities.Event {
+	events := make([]entities.Event, 0, len(c.eventMap))
+	for _, event := range c.eventMap {
+		events = append(events, event)
+	}
+	return events
+}
+
+// GetOptimizelyAudiences returns audiences slice
+func (c DatafileProjectConfig) GetOptimizelyAudiences() []entities.OptimizelyAudience {
+	optimizelyAudiences := make([]entities.OptimizelyAudience, 0, len(c.audienceMap))
+	for _, audience := range c.audienceMap {
+		newOptimizelyAudience := audience.ToOptimizelyAudience()
+		if newOptimizelyAudience.ID == "$opt_dummy_audience" {
```

Comment:
Would it make more sense here to use != then append and avoid using continue?

Reply:
Sure! Thanks!